packages/fxa-payments-server/src/components/AlertBar.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { AlertBar } from './AlertBar';
+
+storiesOf('components/AlertBar', module)
```

Comment:
This is what a basic Storybook story looks like - just a canned rendering of a React component.